test/generateRoutesWithHerbarium.test.js

```diff
@@ -0,0 +1,126 @@
+const assert = require('assert')
+const request = require('supertest')
+
+const { Ok, step, usecase } = require('@herbsjs/buchu')
+const express = require('express')
```

Comment:
do we need this dependencies for tests (express and supertest)? @Anderson-Costa-Vx @jhomarolo 
is it possible to stub it?

Reply:
@dalssoft since we use `supertest` inside other test files to check de integrity and the status code of the endpoint, I don't see a big problem to keep this inside this PR.

I agree with you we should try to remove them, but for me, this is another issue to discuss